test/modules/aws-account-integration.ts

```diff
@@ -121,13 +129,47 @@ describe('AwsAccount Integration Testing', () => {
     })();
   });
 
-  it('inserts a second, useless row into the aws_account table', query(`
-    INSERT INTO aws_account (access_key_id, secret_access_key, region)
-    VALUES ('fake', 'creds', 'us-west-2')
+  it('inserts a second, useless row into the aws_credentials table', query(`
+    INSERT INTO aws_credentials (access_key_id, secret_access_key) VALUES ('fake', 'creds')
   `));
 
   it('does absolutely nothing when you apply this', apply());
 
+  it('selects a default region', query(`
+    UPDATE aws_regions SET is_default = TRUE WHERE region = '${process.env.AWS_REGION}';
+  `));
+
+  it('confirms that the default region was set', query(`
+    SELECT * FROM aws_regions WHERE is_default = TRUE;
+  `, (res: any[]) => expect(res.length).toBe(1)));
+
+  it('tries to set a second default region', (done) => {
+    (async () => {
+      try {
+        await runSql(`
+          UPDATE aws_regions SET is_default = TRUE WHERE region = 'us-east-1';
+        `, false);
+      } catch (_) {
+        return done(); // This is the expected path
+      }
+      return done(new Error('Did not get the expected error'));
+    })();
+  });
+
+  it('confirms that the default region was not changed', query(`
+    SELECT * FROM aws_regions WHERE is_default = TRUE;
+  `, (res: any[]) => {
+    expect(res.length).toBe(1);
+    expect(res[0].region).toBe(process.env.AWS_REGION);
+  }));
+
+  it('updates the default region with the handy `default_aws_region` function', query(`
+    SELECT * FROM default_aws_region('us-east-1');
```

Comment:
I think all the docs changes are coming on later on, but https://docs.iasql.com/aws/ should include the new UI screen and this function?

Reply:
No UI screen changes in the other PR, so I think we're still good?

Reply:
Right, but there are paired PRs? We should update the docs with the new screen that let users select the region is what I mean. I guess it can be another PR other than this one, that is fine.